src/batSet.ml

```diff
@@ -794,31 +995,58 @@ module PSet = struct (*$< PSet *)
     get_cmp (create BatInt.compare) == BatInt.compare
   *)
 
-
+  (* TODO: ensure phys. equality for add remove update map_stdlib filter filter_map_stdlib *)
```

Comment:
Is this TODO item still relevant? It looks like it is now done in your code. (If it's not, then it deserves a clarification of what it means.)

Reply:
no, the comment is obsolete. I will delete it...